HolyshiftService/DB models/HolyShiftUser.cs

```diff
@@ -0,0 +1,17 @@
+public class HolyShiftUser
```

Comment:
Have a folder "Database" with all the DB related class. with a sub-folder "Models" which contains the DbModels